assets/src/hooks/useAssignmentData.js

```diff
@@ -2,7 +2,7 @@ import { gql } from 'apollo-boost'
 import { useQuery } from '@apollo/react-hooks'
 
 const GET_ASSIGNMENT_PLANNING_DATA = courseId => gql`
-{
+query Assignment{
```

Comment:
Setting the operation name to be Assignment so that BE Graph layer can identify which query is being requested.